### PR TITLE
Chore/wp graphql cs

### PR DESCRIPTION
**Pull request by theodesp**
# Description

Merge of https://github.com/wpengine/wp-graphql-content-blocks/pull/126 with some ignores. Commit history is maintained.

# Testing

1. Setup docker-compose
2. Run unit tests
3. Run lint tests

They should all pass

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,13 +12,13 @@
...
     "phpunit/phpunit": "^9.5",
-    "wpengine/wpengine-coding-standards": "dev-develop",
     "yoast/phpunit-polyfills": "^1.0",
     "squizlabs/php_codesniffer": "^3.7",
     "phpstan/phpstan": "^1.10",
     "phpstan/extension-installer": "^1.3",
     "szepeviktor/phpstan-wordpress": "^1.3",
     "axepress/wp-graphql-stubs": "^1.14",
+		"axepress/wp-graphql-cs": "^1.0.0-beta",
     "roave/security-advisories": "dev-latest"
```
Comment (justlevine) [inline composer.json:22]: Gremlins 
```suggestion
    "axepress/wp-graphql-stubs": "^1.14",
    "axepress/wp-graphql-cs": "^1.0.0-beta",
    "roave/security-advisories": "dev-latest"
```

**Comment by justlevine** [comment]
@theodesp I branched this and updated to the latest `wp-graphql-cs` in #163, which includes the latest 3.0.x versions of WPCS/VIPCS, for your review 🙏

**Comment by mindctrl** [comment]
The schema linter detected breaking changes, but it appears to be unrelated to this PR (and maybe related to [WPGraphQL changes](https://github.com/wp-graphql/wp-graphql/releases/tag/v1.19.0)?):

```
Detected the following changes (73) between schemas:
[log] ✖  Field EnqueuedAsset.dependencies changed type from [EnqueuedScript] to [EnqueuedAsset]
[log] ✖  Field EnqueuedStylesheet.dependencies changed type from [EnqueuedScript] to [EnqueuedStylesheet]
```

Otherwise looks good to me.

**Review by mindctrl — APPROVED** [review]

**Comment by justlevine** [comment]
> The schema linter detected breaking changes, but it appears to be unrelated to this PR (and maybe related to [WPGraphQL changes](https://github.com/wp-graphql/wp-graphql/releases/tag/v1.19.0)?):
> 
> ```
> Detected the following changes (73) between schemas:
> [log] ✖  Field EnqueuedAsset.dependencies changed type from [EnqueuedScript] to [EnqueuedAsset]
> [log] ✖  Field EnqueuedStylesheet.dependencies changed type from [EnqueuedScript] to [EnqueuedStylesheet]
> ```
> 
> Otherwise looks good to me.

@mindctrl 🎯. Leaking from https://github.com/wp-graphql/wp-graphql/pull/2988

**Pull request merged**